### PR TITLE
New service pakiti

### PR DESCRIPTION
**Pull request by stavamichal**
- gen script for service pakiti generating users in JSON forma
 - generic send script for pakiti
 - slave script for calling import users as pakiti php library

### CODE REVIEW

File: slave/process-pakiti/bin/process-pakiti.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+			diff -q "${PERUN_FILE}" "${CACHE_FILE}"
+			# no changes, we can log about no changes for this specific config path and skip it
+			if [ $? -eq 0 ]; then
+				log_msg I_NOT_CHANGED
+				continue
+			fi
+		fi
+
+		#let pakiti update it's users for this specific config path
+		${PHP_LIB} ${PAKITI_IMPORT_SCRIPT_PATH} -c import --config="$PAKITI_CONFIG_PATH/Config_[$FROM_PERUN]" < "${PERUN_FILE}"
```
Comment (tauceti2) [inline slave/process-pakiti/bin/process-pakiti.sh]: Please change Config_[$FROM_PERUN] to Config_${FROM_PERUN}.php, so the variable FROM_PERUN doesn't need to contain .php.

Comment (tauceti2) [reply slave/process-pakiti/bin/process-pakiti.sh]: And --config must be the first parameter before -c import.

**Comment by zlamalp** [comment]
@tauceti2 can you please check it again, so it can be merged ? Thank you.

**Comment by tauceti2** [comment]
Checked, it can be merged.

**Pull request merged**